actors/multisig/tests/util.rs

```diff
@@ -43,4 +50,53 @@ impl ActorHarness {
         rt.verify();
         ret
     }
+
+    pub fn propose_ok(
+        self: &Self,
+        rt: &mut MockRuntime,
+        to: Address,
+        value: TokenAmount,
+        method: MethodNum,
+        params: RawBytes,
+    ) -> [u8; 32] {
+        //expect_ok(self.propose(rt, to, value, method, params));
+        rt.expect_validate_caller_type(vec![*ACCOUNT_ACTOR_CODE_ID, *MULTISIG_ACTOR_CODE_ID]);
+        let propose_params =
+            ProposeParams { to: to, value: value.clone(), method: method, params: params.clone() };
+        expect_ok(
+            rt.call::<Actor>(Method::Propose as u64, &RawBytes::serialize(propose_params).unwrap()),
+        );
+        // compute proposal hash
+        let txn = Transaction {
+            to: to,
+            value: value,
+            method: method,
+            params: params,
+            approved: vec![rt.caller],
+        };
+        compute_proposal_hash(&txn, rt).unwrap()
+    }
+
+    pub fn assert_transactions(
```

Comment:
@anorth I've changed the logic to address what seems like an error in the specs-actors method.  specs-actors doesn't care about txn ids and assumes that traversal order will match the expectation slice.  This appears to work only because tests all contain either 0 txns or only 1 unique (thought maybe duplicated) txn.

Reply:
Thanks. This looks ok. I don't yet know enough Rust to tell if there's a better implementation.

If the type-definition overhead is ~zero, it might be slightly nicer to pass a vector of tuples rather than hash map, and then construct a similar vector from the actuals and compare them for equality. But only if you can do that inline, without defining a new type.

Reply:
Did this plus sorting by tx ids before compare